packages/module/realease.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
This all looks good, but I think you've got a typo in the filename that would cause an issue.